README.md

```diff
@@ -10,3 +10,9 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Suppose A has 2 nodes that are connected by an edge, making it an unconnected graph. Now suppose B has 2 nodes that aren't connected either. 
+
+We can have a bijective function that maps each node of A to the nodes of B since they have the same number of nodes and there are no adjacent edges. 
```

Comment:
How do you know that there is such bijective function?

Reply:
Since both the graphs have the same cardinality

Reply:
That alone isn't sufficient.

Reply:
Where can I start in proving a bijective function exists?

Reply:
You can give a way of constructing one. Or maybe it suffices to show an example.